rollingpin/main.py

```diff
@@ -188,8 +188,13 @@ def _main(reactor, *raw_args):
         for host in hosts:
             print host.name
     else:
-        deployer = Deployer(config, event_bus, args.parallel,
-                            args.sleeptime, args.timeout)
+        deployer = Deployer(config,
+                            event_bus,
+                            args.parallel,
+                            args.sleeptime,
+                            args.timeout,
+                            args.dangerously_fast,
+                            )
```

Comment:
🔕 I guess pep8 does support this, but I just find it a bit odd to have that dangling close parenthesis out there.  https://github.com/reddit/styleguide/blob/master/python/pep-0008.rst#indentation  Ignore me though if you like it.